codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -176,15 +175,72 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
         }
     }
 
+    @Override
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
```

Comment:
(And yes, I see they no longer `throws Exception`, but that says nothing about `RuntimeExceptions`, which are the bigger issue.)